packages/solvuu_build/solvuu_build.0.0.1/descr

```diff
@@ -0,0 +1 @@
+Solvuu's build system.
```

Comment:
@Chris00 I finally updated the README for [solvuu-build](https://github.com/solvuu/solvuu-build). Sorry that took so long!